dsgrid/config/dataset_config.py

```diff
@@ -181,14 +182,30 @@ class StandardDataSchemaModel(DSGBaseModel):
 
 
 class OneTableDataSchemaModel(DSGBaseModel):
-    """ data schema model for one table load data format """
-
     load_data_column_dimension: DimensionType = Field(
         title="load_data_column_dimension",
         description="The data dimension for which its values are in column form (pivoted) in the load_data table.",
     )
 
 
+class DatasetQualifierType(DSGEnum):
+    INDEPENDENT = "independent"
```

Comment:
`Independent` (default) as in these datasets represent data that would be taken as an "independent variable" in an equation, contracts the fact that growth_rates are parameters in an equation, and derived_datasets would be like dependent variables/output.

Other terms we can consider are: `input` (as in input variable) or `primary_input`. "growth_rate" would be a process-level dataset in a input->process->output paradigm. Up for other recommendations too.

Reply:
Thanks.

"Quantity" would break down when Metric Class = EnergyEfficiency. And you can still have a growth rate for EnergyEfficiency.

If qualifier is an important discriminator, I think it's easy to give it a default value, because it's easier to check that qualifier == some_value than to check if qualifier exists for a dataset.